### DISCUSSION TITLE
New Guard stack can not be created

### DISCUSSION THREAD

**Discussion by CitizenDroid** [scripthookvdotnet/scripthookvdotnet#1412] [Q&A]
I kept getting this issue which I believe is linked to this mod. I think i may have found a fix at least for my issue I changed my game config as I think its a mod tripping the game stack protection and game config ups the limits set in game for different items maybe its that or that I just  cleaned my memory cache you can use a little cmd app called EmptyStandbyList.exe just google it and run system clean up for directx cache either way I have been playing for a while withj no sign of issue. fingers crossed.

**Comment by kagikn** [comment]
What made you believe the issue is responsible to this SHVDN? I wonder how to reproduce. Also you sure you have sufficient memory (counting both physical or virtual) and no programs or scripts are running with non-trivial memory leak? I'm assuming the issue is about "*A new guard page for the stack cannot be created*" btw.
